x-pack/test/functional/page_objects/remote_clusters_page.ts

```diff
@@ -31,5 +31,31 @@ export function RemoteClustersPageProvider({ getService }: FtrProviderContext) {
       await comboBox.setCustom('comboBoxInput', seedNode);
       await testSubjects.click('remoteClusterFormSaveButton');
     },
+    async getRemoteClustersList() {
+      const table = await testSubjects.find('remoteClusterListTable');
+      const rows = await table.findAllByCssSelector('.euiTableRow');
+      return await Promise.all(
+        rows.map(async (row) => {
+          return {
+            remoteLink: await row.findByTestSubject('remoteClustersTableListClusterLink'),
```

Comment:
This is actually going to return the element, right?  I didn't see remoteLink being used so I don't know if it should really be a link?

Reply:
Yes we return the link with the object for whenever you want to click on the link for that remote cluster.